GooglePay/src/main/java/com/braintreepayments/api/googlepay/GooglePayLauncher.kt

```diff
@@ -0,0 +1,73 @@
+package com.braintreepayments.api.googlepay
+
+import androidx.activity.ComponentActivity
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.ActivityResultRegistry
+import androidx.annotation.VisibleForTesting
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.LifecycleOwner
+
+/**
+ * Responsible for launching the Google Pay payment sheet
+ */
+class GooglePayLauncher @VisibleForTesting internal constructor(
+    registry: ActivityResultRegistry,
+    lifecycleOwner: LifecycleOwner,
+    callback: GooglePayLauncherCallback
+) {
+
+    private val activityLauncher: ActivityResultLauncher<GooglePayPaymentAuthRequestParams> = registry.register(
+        GOOGLE_PAY_RESULT, lifecycleOwner,
+        GooglePayActivityResultContract()
+    ) { googlePayPaymentAuthResult: GooglePayPaymentAuthResult? ->
+        callback.onGooglePayLauncherResult(
+            googlePayPaymentAuthResult
+        )
```

Comment:
```suggestion
        callback.onGooglePayLauncherResult(googlePayPaymentAuthResult)
```
nit: It just feels like the linter is very aggressively auto formatting. It actually has a reverse effect of making the code less readable IMO.